src/reader.go

```diff
@@ -144,6 +150,25 @@ func (r *Reader) readFromStdin() bool {
 	return true
 }
 
+func (r *Reader) readFiles() bool {
+	fn := func(path string, mode os.FileInfo) error {
+		path = filepath.Clean(path)
+		if path != "." {
+			if mode.Mode().IsDir() && filepath.Base(path)[0] == '.' {
+				return filepath.SkipDir
+			}
+			if r.pusher([]byte(path)) {
+				atomic.StoreInt32(&r.event, int32(EvtReadNew))
+			}
+		}
+		return nil
+	}
+	cb := walker.WithErrorCallback(func(pathname string, err error) error {
+		return nil
+	})
+	return walker.Walk(".", fn, cb) == nil
```

Comment:
Thanks @mattn!

Looks like we can just omit error callback here. i.e. `return walker.Walk(".", fn) == nil`

Reply:
Default behavior of walker stop walking when error found. For example, the directory not permitted.

Reply:
Thanks for the clarification.